### DISCUSSION TITLE
Forge无法用mcsl运行

### DISCUSSION THREAD

**Discussion by Leonmmcoset** [MCSLTeam/MCSL2#186] [功能建议] [answered]
用Forge安装服务端后MCSL就没用了！
你要运行run.bat才能运行Forge端，MCSL运行的只是原版
（我是在原版上下载Forge服务器端：用installer）

**Comment by ltzXiaoYanMo** [comment]
？

**Reply by Leonmmcoset** [reply]
?

**Comment by lxhtt** [comment, marked as answer]
在自动安装器没有修好之前 确实只能通过这个脚本 或者说你可以把脚本里面作为JVM参数的内容复制到MCSL里面

**Reply by Leonmmcoset** [reply]
增加@user_jvm_args.txt @libraries/net/neoforged/neoforge/20.4.232/win_args.txt %*
还是前面加java?

**Comment by Leonmmcoset** [comment]
NeoForge在JVM里增加：@user_jvm_args.txt @libraries/net/neoforged/neoforge/20.4.232/win_args.txt %*就行了
（已测试）
